src/main/java/fi/fmi/avi/converter/ConversionHints.java

```diff
@@ -273,6 +273,21 @@ public final class ConversionHints implements Map<Object, Object>, Cloneable {
      */
     public static final Object VALUE_WHITESPACE_SERIALIZATION_MODE_PASSTHROUGH = "WHITESPACE_SERIALIZATION_PASSTHROUGH";
 
+    /**
+     * Controls if zero values for the minute part of point coordinates are dropped (default) or not
+     */
+    public static final Key KEY_SPECIFY_ZERO_MINUTES_IN_COORDINATES;
+
+    /**
+     * Drop zero value for the minute part of point coordinates, i.e. E002 for 2 degrees East
+     */
+    public static final Object VALUE_DROP_ZERO_MINUTES="DROP_ZERO_MINUTES";
+
+    /**
+     * Always specify zero value for the minute part of point coordinates, i.e. E00200 for 2 degrees East (for KNMI SIGMET/AIRMET)
+     */
+    public static final Object VALUE_SPECIFY_ZERO_MINUTES="SPECIFY_ZERO_MINUTES";
```

Comment:
Naming convention for values seem to have been `VALUE_<KEY_NAME>_<VALUE_NAME>`. I would suggest following names (please develop further):

```suggestion
    /**
     * Controls whether the minutes part of point coordinates is omitted (default) or not when minutes is zero.
     */
    public static final Key KEY_COORDINATE_MINUTES;

    /**
     * Omit the minutes part of point coordinates when minutes is zero (default). I.e. E002 for 2 degrees East.
     */
    public static final Object VALUE_COORDINATE_MINUTES_OMIT_ZERO = "OMIT_ZERO";

    /**
     * Include the minutes part of point coordinates when minutes is zero. I.e. E00200 for 2 degrees East.
     */
    public static final Object VALUE_COORDINATE_MINUTES_INCLUDE_ZERO = "INCLUDE_ZERO";
```

Reply:
Names have been changed